app/helpers/application_helper.rb

```diff
@@ -35,7 +35,7 @@ def get_nav_items
       countries: [
         {
           title: I18n.t("countries.#{yml_key}.title"),
-          url: country_path(54)
+          url: country_path(57)
```

Comment:
I think I would already start using ISO codes at least as those are better unique keys then ids in this case.
Development environments can break easily because ids can be different depending on how countries have been imported.

Also I can't really see an advantage of using ids even from a user perspective

Reply:
Sure thing @Levia I will switch to using ISO3 codes in this PR.